express-hello-world/server.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+const app = express();
+app.listen(8080, () => {
```

Comment:
We usually put the `.listen` call at the very end of the file. It's just convention and I don't think it makes a difference, but I suggest you get into the habit, just in case! The idea is that you want to make all the calls that configure express first, then start listening.